src/views/serviceReport/ServiceReport.jsx

```diff
@@ -159,54 +201,95 @@ class ServiceReport extends Component {
-                      <i className="fas fa-file-download"></i> PDF
-                    </Button>
-                    <UncontrolledTooltip placement="top" target={'down_pdf'}>
-                      Download as PDF
-                    </UncontrolledTooltip>
-                  </span>
+                  <Row>
+                    <Col md="5">
+                      <h3 className="mb-0">Service Report</h3>
+                    </Col>
+                    <Col md="4">
+                      <Label for="startdate">Start Date</Label>
+                      <DatePicker
+                        selected={this.state.startDate}
+                        onChange={this.sdateChange}
+                        dateFormat="yyyy-MM-dd"
+                        placeholderText="Select Start Date"
+                      />
+                      <Label for="endtdate">End Date</Label>
+                      <DatePicker
+                        selected={this.state.endDate}
+                        onChange={this.edateChange}
+                        dateFormat="yyyy-MM-dd"
+                        placeholderText="Select End Date"
+                      />
+                      <div className="flex" style={{ marginTop: '32px' }}>
+                        <Button
+                          color="primary"
+                          id="FilterTooltip"
+                          onClick={this.onClickFilter}
+                        >
+                          {' '}
+                          <i className="fas fa-filter"></i>
+                        </Button>
+                        <UncontrolledTooltip
+                          placement={'top'}
+                          target={'FilterTooltip'}
+                        >
+                          Filter
+                        </UncontrolledTooltip>
+
+                        <Button
+                          color="primary"
+                          style={{ marginLeft: '10px' }}
+                          onClick={this.resetFilter}
+                          id="reset_tool"
+                        >
+                          <i className="fas fa-history"></i>
+                        </Button>
+                        <UncontrolledTooltip
+                          placement={'top'}
+                          target={'reset_tool'}
+                        >
+                          Reset
+                        </UncontrolledTooltip>
+                      </div>
+                    </Col>
+                    <Col md="3">
+                      <Button
+                        color="primary"
+                        size="sm"
+                        onClick={this.download}
+                        id="down_csv"
+                      >
+                        <i className="fas fa-file-download"></i> CSV
+                      </Button>
+                      <CSVLink
+                        data={this.state.dataToDownload}
+                        filename={'Service Report' + '.csv'}
```

Comment:
pls add current date string to end of file name. construct file name in render function and add here